api/prisma/schema.prisma

```diff
@@ -127,7 +125,10 @@ model user {
   portfolio                    Portfolio[]
   profileUI                    ProfileUI? // Undefined
   progressTimestamps           Json? // ProgressTimestamp[] | Null[] | Int64[] | Double[] - TODO: NORMALIZE
-  // rand Float? // Undefined removed, as new API never uses it.
+  /// A random number between 0 and 1.
+  ///
+  /// Valuable for selectively performing random logic.
+  rand                         Float?
```

Comment:
This is necessary, because we need to document "odd" things in the user record, lest I, in 1 years time, wonder "why do we have this property we do not use", and remove it? _Like I almost did_

Reply:
I'm really not keen on adding things we don't have a clear use for.

Is the point just that we haven't, yet, removed this from all users so we want a record of that fact? The concern being we might forget about this, add `rand` back to the schema later and get a nasty surprise?

Reply:
Both. We have the property, and we specifically discussed not removing it during the normalization. So, it should be in the schema, even though it is not used currently in the api, because we do expect to use it.

Reply:
Most documents have it. It is supposed to be use for things like A/B tests.

Reply:
Eh, we already have A/B tests without it.

Anyways, it's only clutter. If we never use it, it's not the end of the world.